reactioncommerce/reaction-component-library
# (feat): InPageMenu and InPageMenuItem components

Resolves #66 
Impact: **minor**  
Type: **feature**

## Component
This PR adds the InPageMenu and InPageMenuItem components, for use as sub-navigation menus inside of pages.

## Screenshots
![inpagemenu_ _reaction_design_system](https://user-images.githubusercontent.com/4482263/46823910-5148dd00-cd44-11e8-847c-1101d76b606b.png)

## Breaking changes
None

## Testing
- See that the InPage Menu houses InPageMenuItems correctly
- See that `isSelected` styles are applied to InPageMenuItem correctly
- See that href / onClick works correctly
